kerastuner/__init__.py

```diff
@@ -30,6 +30,7 @@
 from .tuners import BayesianOptimization
 from .tuners import Hyperband
 from .tuners import RandomSearch
+from .tuners import Sklearn
```

Comment:
Thought here: "Sklearn" is not a search algorithm (unlike the others, like `RandomSearch`). Also, it may ause confusion vs. the package name, `sklearn`. I'd recommend using the suffix: `SklearnTuner`.